geoh5py/objects/drillhole.py

```diff
@@ -707,8 +616,8 @@ def validate_depth_data(
 
         return values
 
-    def validate_data(
-        self, attributes: dict, property_group=None, collocation_distance=None
+    def validate_association(
+        self, attributes: dict, property_group=None, collocation_distance=None, **_

```

Comment:
What's going on here?

Reply:
Same as for the other one, this is just a wild card to allow for variable kwargs in the sub-classes

Reply:
Is validate_association declared an abstractmethod further up in the hierachy?  If not, you should be free to add keyword arguments in child class definitions without this annotation, right?